compute_tools/src/http/openapi_spec.yaml

```diff
@@ -68,6 +68,45 @@ paths:
               schema:
                 $ref: "#/components/schemas/Info"
 
+  /schema/objects:
```

Comment:
NIT: I don't think that roles and databases can be considered as schema, they are still records within some schema, so I'd actually separate these two routes into

```
/dbs_and_roles

DatabasesAndRoles: ...
Database: ...
Role: ...
GenericOption: ...
```

and

```
/schema
```

Reply:
> they are still records within some schema

so, they are part of the schema, aren't they? :)

Reply:
Well, to me -- not, schema is schema and roles / dbs are records, effectively data, just in the shared catalog, but maybe I overthink it :)

IIRC, `pg_dump --schema-only` won't dump roles and dbs, because they are not considered as a part of schema. Schema is kinda db-level thing, while roles and dbs are above

I don't know, I was just confused seeing dbs / roles nearby schema

Reply:
@tristan957 WDYT?

Reply:
I think if Alexey's comment is right:

> IIRC, pg_dump --schema-only won't dump roles and dbs, because they are not considered as a part of schema. Schema is kinda db-level thing, while roles and dbs are above

Then we should move the `/objects` endpoint out of the /schema` endpoint.

Reply:
Ok, how about this:

I renamed `/schema/objects` to `catalog/objects` and `/schema/ddl` to `/catalog/database_ddl`. Does it look like more correct naming? WDYT?

Reply:
Sorry, don't really want to do bikeshedding, but I'd
- replace `/catalog/database_ddl` with just `/schema` or `/database_schema`. It's more straightforward and clean. I think in Postgres catalog or shared catalog or pg_catalog is usually used to refer shared tables https://www.postgresql.org/docs/current/catalogs.html and it's not what we get here
- `catalog/objects` is probably OK, but sounds a bit too broad, so I really don't see any cons in just naming it more specific like `/dbs_and_roles`, do you?

Reply:
> I think in Postgres catalog or shared catalog or pg_catalog is usually used to refer shared tables https://www.postgresql.org/docs/current/catalogs.html and it's not what we get here

yes! And pg_dump --schema-only is basically a utility that transforms pg_catalog entities into DDL statements, isn't it? 

@tristan957 please, make your choice :)

Reply:
I'll defer to Alexey :). Thanks for your work on this thus far.